dev-infrastructure/configurations/mgmt-cluster.tmpl.bicepparam

```diff
@@ -43,4 +41,4 @@ param mgmtKeyVaultName = '{{ .mgmtKeyVault.name }}'
 param aroDevopsMsiId = '{{ .aroDevopsMsiId }}'
 
 // Azure Monitor Workspace
-param azureMonitorWorkspaceName = '{{ .monitoring.workspaceName }}'
+param azureMonitoringWorkspaceId = '__azureMonitoringWorkspaceId__'
```

Comment:
What is `__azureMonitoringWorkspaceId__` a variable?

Reply:
it is an output chaining variable. so it is filled with an output value from another bicep run